analyzer/src/main/kotlin/managers/Unmanaged.kt

```diff
@@ -69,17 +69,15 @@ class Unmanaged(
             vcsInfo == VcsInfo.EMPTY -> {
                 // This seems to be an analysis of a local directory that is not under version control, i.e. that is not
                 // a VCS working tree. In this case we have no change to get a version.
-                val projectDir = definitionFile
-
                 log.warn {
-                    "Analysis of local directory '$projectDir' which is not under version control will produce " +
+                    "Analysis of local directory '$definitionFile' which is not under version control will produce " +
```

Comment:
FYI: The variable was actually introduced intentionally to make it explicit that in this case not a *file* but a *dir* is being handled. But maybe the mention of "directory" in the warn log is enough.

Reply:
I would think so, as it's also explained in the code comment.